flink-python/src/main/java/org/apache/flink/table/runtime/operators/python/aggregate/arrow/batch/BatchArrowPythonGroupWindowAggregateFunctionOperator.java

```diff
@@ -175,6 +175,7 @@ public void emitResult(Tuple2<byte[], Integer> resultTuple) throws Exception {
             windowAggResult.replace(key, arrowSerializer.read(i));
             rowDataWrapper.collect(reuseJoinedRow.replace(windowAggResult, windowProperty));
         }
+        arrowSerializer.reinitializeReader(bais);
```

Comment:
Is it possible to move `reinitializeReader(bais)` inside `ArowSerializer.load()`. Then we could avoid this change.